indexer/docker-compose.yml

```diff
@@ -16,33 +16,24 @@ services:
     volumes:
       - postgres_data:/data/postgres
 
-  indexer:
+  api:
     build:
       context: ..
-      dockerfile: indexer/Dockerfile
+      dockerfile: indexer/Dockerfile.refresh
+    command: ["indexer-refresh", "api"]
     healthcheck:
       test: wget localhost:8080/healthz -q -O - > /dev/null 2>&1
-    environment:
-      # Note that you must index goerli with INDEXER_BEDROCK=false first, then
-      # reindex with INDEXER_BEDROCK=true or seed the database
-      - INDEXER_BEDROCK=${INDEXER_BEDROCK_GOERLI:-true}
-      - INDEXER_BUILD_ENV=${INDEXER_BUILD_ENV:-development}
-      - INDEXER_DB_PORT=${INDEXER_DB_PORT:-5432}
-      - INDEXER_DB_USER=${INDEXER_DB_USER:-db_username}
-      - INDEXER_DB_PASSWORD=${INDEXER_DB_PASSWORD:-db_password}
-      - INDEXER_DB_NAME=${INDEXER_DB_NAME:-db_name}
-      - INDEXER_DB_HOST=${INDEXER_DB_HOST:-postgres}
-      - INDEXER_CHAIN_ID=${INDEXER_CHAIN_ID:-5}
-      - INDEXER_L1_ETH_RPC=$INDEXER_L1_ETH_RPC
-      - INDEXER_L2_ETH_RPC=$INDEXER_L2_ETH_RPC
-      - INDEXER_REST_HOSTNAME=0.0.0.0
-      - INDEXER_REST_PORT=8080
-      - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=0
-      - INDEXER_BEDROCK_L1_STANDARD_BRIDGE=${INDEXER_BEDROCK_L1_STANDARD_BRIDGE:-0x636Af16bf2f682dD3109e60102b8E1A089FedAa8}
-      - INDEXER_BEDROCK_OPTIMISM_PORTAL=${INDEXER_BEDROCK_OPTIMISM_PORTAL:-0xB7040fd32359688346A3D1395a42114cf8E3b9b2}
-      - INDEXER_L1_ADDRESS_MANAGER_ADDRESS=${INDEXER_L1_ADDRESS_MANAGER_ADDRESS:-0xdE1FCfB0851916CA5101820A69b13a4E276bd81F}
     ports:
       - 8080:8080
+    environment:
+      - INDEXER_DB_PORT=5432
+      - INDEXER_DB_USER=db_username
+      - INDEXER_DB_PASSWORD=db_password
+      - INDEXER_DB_NAME=db_name
+      - INDEXER_DB_HOST=postgres
+      - INDEXER_CONFIG=/configs/indexer.toml
```

Comment:
what part of the indexer config does the api need? Do you think the api module will need access access the rpc urls?